DocumentDataAPI/DocumentDataAPI/Data/Repositories/NpgWordRatioRepository.cs

```diff
@@ -15,12 +15,12 @@ public NpgWordRatioRepository(IDbConnectionFactory connectionFactory, ILogger<Np
         _logger = logger;
     }
 
-    public WordRatioModel Get(int id)
+    public WordRatioModel? Get(long id)
     {
         _logger.LogDebug("Retrieving WordRatio with id {id} from database", id);
         using IDbConnection con = _connectionFactory.CreateConnection();
-        return con.QuerySingle<WordRatioModel>("select * from word_ratios " +
-                                               "where id=@Id", new { id });
+        return con.Query<WordRatioModel>("select * from word_ratios where id=@Id", new {id})
```

Comment:
Er det her bare til at force den til at return null, hvis ikke der er elements? Det virker lidt sus, at tvinge den til det, bare for at tjekke for null senere.

Reply:
`QuerySingle` forventer at der er netop ét ikke-null resultat fra querien, og kaster derfor en exception hvis det ikke er tilfældet. `Query` returnerer objektet hvis det findes, og null ellers. Jeg har forklaret hvorfor det er smart i kommentaren ovenfor